lib/Models/GltfCatalogItem.ts

```diff
@@ -99,19 +101,27 @@ export default class GltfCatalogItem
     }
     this._model.show = this.show;
 
-    let position;
-    if (this.origin !== undefined) {
+    let position: Cartesian3;
+    if (
+      this.origin !== undefined &&
+      this.origin.longitude !== undefined &&
+      this.origin.latitude !== undefined &&
+      this.origin.height !== undefined
+    ) {
       position = Cartesian3.fromDegrees(
-        this.origin.longitude || 0,
-        this.origin.latitude || 0,
-        this.origin.height || 0
+        this.origin.longitude,
+        this.origin.latitude,
+        this.origin.height
       );
+    } else {
+      position = Cartesian3.ZERO;
     }
 
-    const dataSource = new CustomDataSource(this.name || "glTF model");
+    const dataSource: CustomDataSource = new CustomDataSource(this.name || "glTF model");
     dataSource.entities.add(
       new Entity({
-        position,
+        position: new ConstantPositionProperty(position),
+        orientation: Quaternion.IDENTITY as any,
```

Comment:
Sometimes! In some cases Cesium puts stuff at weird orientations if you don't specify the orientation to the identity. @kring would have a better idea why...

Reply:
Yeah the default is east-north-up or somesuch, rather than ECEF. But better than that dodgy cast (which is totally my fault btw) would be:
`orientation: new ConstantPositionProperty(Quaternion.IDENTITY)`

Reply:
```suggestion
        orientation: new ConstantPositionProperty(Quaternion.IDENTITY),
```